lib/Checkout/Payments/Request/Source/Apm/RequestSepaSource.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Checkout\Payments\Request\Source\Apm;
+
+use Checkout\Common\AccountHolder;
+use Checkout\Common\Address;
+use Checkout\Common\PaymentSourceType;
+use Checkout\Payments\Request\Source\AbstractRequestSource;
+
+class RequestSepaSource extends AbstractRequestSource
+{
+    public function __construct()
+    {
+        parent::__construct(PaymentSourceType::$sepa);
+    }
+
+    /**
+     * @var Country
```

Comment:
Please check the SDK it should be `* @var string values of Country`

Reply:
Ok 👍